identity/src/main/java/com/android/identity/credential/CredentialUtil.java

```diff
@@ -107,7 +107,7 @@ public static int managedAuthenticationKeyHelper(
             for (int n = 0; n < numNonReplacementsToGenerate; n++) {
                 Credential.PendingAuthenticationKey pendingKey =
                         credential.createPendingAuthenticationKey(createKeySettings, null);
-                pendingKey.setApplicationData(managedKeyDomain, new byte[0]);
+                pendingKey.getApplicationData().setData(managedKeyDomain, new byte[0]);
             }
         }
```

Comment:
This code would be clearer if we use `getBoolean()` / `setBoolean()` here. This is technically a break but since `managedAuthenticationKeyHelper()` is brand new code I think we can do this (if it had been around for a while and in use in the field this would be a no-no since it would break existing wallets using this).